onnxruntime/core/framework/data_transfer.cc

```diff
@@ -9,6 +9,14 @@ common::Status IDataTransfer::CopyTensor(const Tensor& src, Tensor& dst) const {
   return CopyTensor(src, dst, 0);
 }
 
+common::Status IDataTransfer::CopyTensors(const Tensor* src, Tensor* dst, int size) const {
+  ORT_ENFORCE(nullptr != src && nullptr != dst);
```

Comment:
Also enforce size > 0?

Reply:
it should be ok. <0 input will not hurt this function though it's a useless call.